packages/ilios-common/addon/components/offering-manager.gjs

```diff
@@ -40,33 +42,66 @@ export default class OfferingManagerComponent extends Component {
     return new TrackedAsyncData(this.args.offering.learnerGroups);
   }
 
-  @cached
-  get sessionData() {
-    return new TrackedAsyncData(this.args.offering?.session);
+  get learnerGroups() {
+    return this.learnerGroupsData.isResolved ? this.learnerGroupsData.value : null;
   }
 
-  @cached
-  get courseData() {
-    return new TrackedAsyncData(this.session?.course);
+  get sortedLearnerGroups() {
+    if (!this.learnerGroups) {
+      return [];
+    }
+    return this.learnerGroups.toSorted((learnerGroupA, learnerGroupB) => {
+      if ('title:desc' === this.sortBy) {
+        return learnerGroupB.title.localeCompare(learnerGroupA.title, this.intl.primaryLocale, {
+          numeric: true,
+        });
+      }
+      return learnerGroupA.title.localeCompare(learnerGroupB.title, this.intl.primaryLocale, {
+        numeric: true,
+      });
+    });
   }
 
   @cached
-  get cohortsData() {
-    return new TrackedAsyncData(this.course?.cohorts);
+  get individualLearnersData() {
+    return new TrackedAsyncData(this.args.offering.learners);
   }
 
-  get learnerGroups() {
-    return this.learnerGroupsData.isResolved ? this.learnerGroupsData.value : null;
+  get individualLearners() {
+    return this.individualLearnersData.isResolved ? this.individualLearnersData.value : [];
+  }
+
+  get sortedIndividualLearners() {
+    if (!this.individualLearners) {
```

Comment:
empty lists are not falsy, and i'm not seeing a falsy value like null or undefined sneaking through here.

```suggestion
    if (!this.individualLearners.length) {
```